src/test/osd/osd-scrub-repair.sh

```diff
@@ -32,7 +32,18 @@ function run() {
     local dir=$1
     shift
 
-    export CEPH_MON="127.0.0.1:7107" # git grep '\<7107\>' : there must be only one
+    # Grab a random port from 7109 - 7200, this should maybe be reworked
+    for port in `seq 7109 7200 | shuf` ; do
+        if ! $(nc -z localhost "$port" > /dev/null); then
```

Comment:
yeah it is racey, thanks for the comments, I like your fix for using a port tied to the test number. Using a fixed port per script has problems when trying to run things concurrently, but if the port is based on the test #, each test # is unique and that avoids having problems. I assume the test number is probably exposed as a environment variable or something, I look into that. Thanks!